perun-core/src/main/java/cz/metacentrum/perun/core/bl/AttributesManagerBl.java

```diff
@@ -216,6 +216,25 @@
 	List<Attribute> getAttributes(PerunSession sess, Member member, Group group, boolean workWithUserAttributes) throws InternalErrorException, WrongAttributeAssignmentException;
 
 	/**
+	 * Get all attributes associated with the member in the group which have name in list attrNames.
+	 * If workWithUserAttributes is true, get also all <b>non-empty</b> user and member attributes
```

Comment:
Right now javadoc is not consistent with implementation. In case of ``workWithUserAttributes==false`` it returns member-group + member attributes, but javadoc state:  _"If workWithUserAttributes is true, get also all <b>non-empty</b> user and member attributes"_, which means, by default it should return only member-group attributes.

Based on API layer javadoc implementation is correct, so you should update javadoc in Bl layer to match it.